app/frontend/src/components/ContributorForm/constants.ts

```diff
@@ -0,0 +1,80 @@
+export const CONTRIBUTE = "Sign up to volunteer with Couchers.org";
+export const JOIN_THE_TEAM =
+  "Join the worldwide team of couch surfers bringing this community-run platform to life";
+export const FILL_IN_THE_FORM =
+  "Fill in the form below to sign up to volunteer. We'll get in touch if we need your skills.";
```

Comment:
This is in the signup form now so some of these labels need to be different. Can probably remove this one completely.